documentation/modules/appendix/pages/best-practices.adoc

```diff
@@ -88,6 +88,11 @@ When trying to refer to a physical unit, please see the
 xref:appendix:unitcatalog.adoc[Unit Catalog]. When searching for the unit,
 remember that the unit catalog uses British English.
 
+TIP: If you're modeling quantities for which both metric (SI) and imperial units exist, such as
```

Comment:
Line 89: remember that the unit catalog uses British English, e.g. unit:metre instead of meter.

TIP: If you're modeling quantities for which both the metric system and the imperial system provide units, such as meter vs feet, it is always recommended to use the metric system (preferably SI units like meter or others like km if more common in the domain) – unless there are specific reasons to create the model differently. In any case it is strongly discouraged to add multiple Properties in the same scope representing the same value only using different units due to the inherent complexity.